backend-api/src/functions/asyncBiometricToken/asyncBiometricTokenHandler.ts

```diff
@@ -7,17 +7,30 @@ import {
   dependencies,
   IAsyncBiometricTokenDependencies,
 } from "./handlerDependencies";
-import { notImplementedResponse } from "../common/lambdaResponses";
+import {
+  badRequestResponse,
+  notImplementedResponse,
+} from "../common/lambdaResponses";
+import { validateRequestBody } from "./validateRequestBody/validateRequestBody";
 
 export async function lambdaHandlerConstructor(
   dependencies: IAsyncBiometricTokenDependencies,
-  _event: APIGatewayProxyEvent,
+  event: APIGatewayProxyEvent,
   context: Context,
 ): Promise<APIGatewayProxyResult> {
   const logger = dependencies.logger();
   logger.addContext(context);
   logger.log("STARTED");
 
+  const validateRequestBodyResult = validateRequestBody(event.body);
+  if (validateRequestBodyResult.isError) {
+    logger.log("REQUEST_BODY_INVALID", {
+      errorMessage: validateRequestBodyResult.value.errorMessage,
+    });
+    return badRequestResponse("invalid_request", "Request body invalid");
```

Comment:
Nitpick: since we already have more useful information about the validation failure (in `errorMessage`), I wonder if we should just send that back to the client?

Reply:
I did originally do that but then reverted as I didn't know whether it would be worth emitting info such as sessionId and documentType values here. 
- Would clients ever find use in this? 
- If not, we have that info in the logs, do we also need it here?

Reply:
That makes sense to me, updated